docs/adr/adr-0023-adding-stylelint.md

```diff
@@ -0,0 +1,220 @@
+---
+kind: '📌 Docs/Architecture Decision Records'
+---
+# ADR 0023: Adding Stylelint
+
+🗓️ 2023-XX-YY · ✍️ Mathieu Degand
+
+## The context
+
+Since the beginning of this component library we didn't have a dedicated tool to format our CSS and enforce some rules. 
+Yet, we still had a custom ESLint plugin to sort our CSS declarations but that was pretty much it as referenced in [ADR 11](link_adr). 
+
+However, in 2020 we found that Stylelint was able to handle CSS in template literals and that we might be able to use it to replace our ESLint sort rule and enforce some CSS rules to enforce code style and coherence across our library code base.
```

Comment:
Double usage of "enforce" in the same sentence :wink:
The sentence is a bit long.